springwolf-core/src/main/java/io/github/springwolf/core/asyncapi/scanners/common/utils/DescriptionUtil.java

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.springwolf.core.asyncapi.scanners.common.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+
+/**
+ * Auxiliary class to manage Description or long texts.
+ */
+public class DescriptionUtil {
```

Comment:
Have you considered naming it TextUtil? I think the functionality is independent of descriptions.